programs/rewards/src/state/mining.rs

```diff
@@ -98,14 +100,15 @@ pub struct Mining {
     pub unclaimed_rewards: u64,
     /// This field sums up each time somebody stakes to that account as a delegate.
     pub stake_from_others: u64,
-    /// Saved bump for mining account
-    pub bump: u8,
+    /// The date when batch minting is restricted until.
+    pub batch_minting_restricted_until: u64,
     /// Account type - Mining. This discriminator should exist in order to prevent
     /// shenanigans with customly modified accounts and their fields.
     /// 0: account type
     /// 1: claim is restricted
-    /// 2-6: unused
-    pub data: [u8; 7],
+    /// 2: bump
+    /// 3-15: unused
+    pub data: [u8; 16],
```

Comment:
You could actually introduce the named fields directly, instead of using it through data